### ISSUE TITLE
PSR option

### ISSUE THREAD

**Issue by Shnilj** [Shaked/sublime-phptools#13]
Hi,

I have noticed that whenever I enable the "psr" setting, the '*-tmp'-file does not get deleted and the base-file does not get formatted.

Could this be a bug in the plugin or is it something in my system that is making it bug?

Thanks in advance!

**Comment by udokmeci** [comment]
Yes same problem with @Shnilj, the -tmp files are annoying. I had to remove package. This is a huge problem for me.
